### ISSUE TITLE
Ignored files make compilation fail

### ISSUE THREAD

**Issue by andreat** [ultimate-pa/ultimate#370]
Dear developers,

I am trying to use Ultimate in Eclipse, but it fails while building it by showing the same errors on multiple projects. All errors are like "ForkStatement cannot be resolved to a type" and "The import de.uni_freiburg.informatik.ultimate.boogie.ast.ForkStatement cannot be resolved" (or with JoinStatement instead of ForkStatement).
Problems such the above ones occur for instance in /Library-BoogieAST/src/de/uni_freiburg/informatik/ultimate/boogie/BoogieVisitor.java and in /BoogiePreprocessor/src/de/uni_freiburg/informatik/ultimate/boogie/preprocessor/TypeChecker.java.

The issue seems to be related to the fact that {Fork|Join}Statement.java files have been added to the .gitignore file and they are available at build time.

Currently I have Ultimate at revision b4587f3664

Regards,
Andrea

**Comment by danieldietsch** [comment]
Hi Andrea, 
did you check [our troubleshooting section](https://github.com/ultimate-pa/ultimate/wiki/Troubleshooting#ultimate-does-not-compile) for this issue? 

In particular if you setup the development environment for the first time, this might happen.

**Comment by andreat** [comment]
Hi Daniel,

the environment was set up already some time ago and it was working. Then I updated Ultimate and refreshed everything but I was facing the reported problem.

Now it has been solved by making a full clean&build followed by a refresh of Crocotta since after the build induced by the clean, Crocotta needed another refresh to compile correctly. 

I wonder why the simple refresh was not enough. Anyway not it is solved.

Regards,
Andrea
